VCFv4.3.tex

```diff
@@ -94,8 +94,17 @@ \subsection{Character encoding, non-printable characters and characters with spe
 
 
 \subsection{Data types}
-Data types supported by VCF are: Integer (32-bit, signed), Float (32-bit, formatted to match the regular expression \verb|^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$|, \texttt{NaN}, or \texttt{+/-Inf}), Flag, Character, and String.
-For the Integer type, the values from $-2^{31}$ to $-2^{31}+7$ cannot be stored in the binary version and therefore are disallowed in both VCF and BCF, see \ref{BcfTypeEncoding}.
+The data types supported by VCF are:
+
+\begin{itemize}
+  \item Integer (32-bit, signed): Values from $-2^{31}$ to $-2^{31}+7$ cannot be stored in the binary version and therefore are disallowed in both VCF and BCF, see \ref{BcfTypeEncoding}.
+  \item Float (32-bit IEEE-754): Formatted to match one of the regular expressions \verb|^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$| or \verb"^[-+]?(INF|INFINITY|NAN)$" case insensitively.
```

Comment:
```suggestion
  \item Float (32-bit IEEE-754): Formatted to match one of the regular expressions  \verb"^[-+]?(INF|INFINITY|NAN)$" or \verb|^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$| case insensitively.
```
(so that the text doesn't fall out of the margin.)

Reply:
Numeric is the common case and ∞/NaN is the corner case, so swapping them around would not be ideal exposition. (And “case insensitively” pertains to the ∞/NaN regex in particular.)

Reply:
We could join them into one regular expression and put that on it's own line.

Reply:
I tried that, but considered that it was more legible as two separate regexps. YMMV.

The real problem here is that a monster regexp is far from an ideal way to say “floating point number text goes here” — see also https://github.com/samtools/hts-specs/issues/89#issuecomment-111417134.